draft-02.xml

```diff
@@ -969,10 +969,25 @@
             'safe', 'unsafe', or 'idempotent.'
           </t>
           <t>
-            The 'rt' attribute is OPTIONAL and is an opaque string and MAY match the
-            <xref target="prop-id" format="title"/>
-            of an
-            existing<xref target="prop-descriptor" format="title"/>.
+            The 'rt' attribute is OPTIONAL and SHOULD be one of the following:
+            <list style="numbers">
+              <t>
+                A fragment identifier (e.g. rt="#user") which points
+                to the <xref target="prop-id" format="title"/>
+                of an existing <xref target="prop-descriptor" format="title"/>
+                in the current ALPS document.
+              </t>
+              <t>
+                A resolvable URL (e.g. rt="http://example.org/profiles/user")
```

Comment:
just to be clear, we are not just going to say it MUST refer to a descriptor ID in a valid ALPS document (local or external) we as ALSO going to say the value MUST contain a fragment identifier. no other format will will be acceptable now and forever, right? this is what i'm on about here. 

pretty sure some URNs have resolvers. i know there is work on adding something like fragment identifiers to URNs (not sure of details).

i am ok w/ the first part ("MUST refer to a descriptor id in a valid ALPS doc"). 

now i am pushing on the MUST contain a fragment identifier.

you ok w/ both MUSTs?